app/src/main/resources/organizations.yaml

```diff
@@ -0,0 +1,8 @@
+- name: report-stream
+  description: Report Stream
+  senders:
+    - name: default
+      topic: etor
+      customerStatus: active
+      format: FHIR
```

Comment:
I don't think we need to specify format because this TI application will only ever accept FHIR.  I'm not saying we wouldn't ever accept another format, but if we were, some other service (like ReportStream, for example) would convert it first to FHIR and then send it to this TI application.

Reply:
Fair. I'm ok with removing it